.travis.yml

```diff
@@ -7,15 +7,15 @@ script:
 
 # Docker Run to Execute Unit Test Package
 - echo "Running Unit Tests"
-- docker run -e APP_KEY -e APP_TOKEN -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo todo-app:test Test/view_model_test.py
+- docker run -e APP_KEY -e APP_TOKEN -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS todo-app:test Test/view_model_test.py
```

Comment:
We shouldn't be using the APP_KEY etc any longer, so we should remove those as secrets & stop passing them through